rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
Sorry, yes you do need dHrxn.  If we default it to dHrxn=0.0 we just have to replace Ea with Ea-alpha*dHrxn and it works for both cases.  
I agree that it would be an almost-impossible-to-use feature within the RMG context, but I know from experience that RMG users may use our code in ways we didn't intend and it shouldn't take more than a few seconds for us to add this functionality.